.github/workflows/helm-docs.yaml

```
@@ -0,0 +1,164 @@
+name: "Helm Docs Check"
+
+on:
+  pull_request:
+    paths:
+      - "**/*/values.yaml"
+      - "**/*/*/values.yaml"
+
+jobs:
+  helm-docs-check:
+    runs-on: [default]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set up go environment
+        uses: actions/setup-go@v5
+        with:
+          go-version: 1.22.0
```

I would use the latest stable go version and not an old one 
```suggestion
        with:
          go-version: 'stable'
```

---

Thank you. Fixed in e149e6eb5b38a27ff1246bd0ba852e48cc7ea754